spk/jellyfin/Makefile

```
@@ -1,7 +1,7 @@
 # Remember to also update jellyfin-web
 SPK_NAME = jellyfin
-SPK_VERS = 10.8.0
-SPK_REV = 5
+SPK_VERS = 10.8.3
+SPK_REV = 7
```

We are not so fast :)

```suggestion
SPK_REV = 6
```